docs/galaxy.rst

```diff
@@ -48,6 +48,7 @@ The following models are found in the `skypy.galaxy.redshift` package.
 .. autosummary::
    :nosignatures:
 
+   herbel_pdf
```

Comment:
```suggestion
```
`herbel_pdf` is only used internally in `herbel_redshift` and so should not be added to the documentation.